Makefile.am

```diff
@@ -217,10 +217,11 @@ install-exec-local: install_python_embedded_mods stamps/install_ruby_deps_local
 uninstall-local:
 	rm -rf ${DESTDIR}/$(PCS_BUNDLED_DIR)
 	if test "${DESTDIR}/$(prefix)/bin/" != "${DESTDIR}/$(SBINDIR)/"; then \
-		find ${DESTDIR}/$(prefix)/local/sbin -name pcs -type f -delete || \
-			rm -fv ${DESTDIR}/$(SBINDIR)/pcs; \
-		find ${DESTDIR}/$(prefix)/local/sbin -name pcsd -type f -delete || \
-			rm -fv ${DESTDIR}/$(SBINDIR)/pcsd; \
+		if ls ${DESTDIR}/$(prefix)/local/sbin/pcs; then \
+			rm -fv ${DESTDIR}/$(prefix)/local/sbin/pcs*; \
```

Comment:
We don't want to delete e.g. pcsc by mistake, specify exactly pcs and pcsd, in both if and else branches